app/helpers/component_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ComponentHelper
+  def add_option(option, output = nil)
```

Comment:
Can we change this method to
```ruby
  def add_option(option, output = nil)
    return unless option
    return output if output
    option
  end
```
So that it is easier to read?